app/konnect/org-management/azure-idp.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Set up SSO with Azure
+content_type: how-to
+---
+
+As an alternative to Kong Konnect’s native authentication, you can set up single sign-on (SSO) access to Konnect through an identity provider (IdP) with OpenID Connect. This authentication method allows your users to log in to Kong Konnect using their IdP credentials, without needing a separate login.
+
+You can’t mix authentication methods in Kong Konnect. With IdP authentication enabled, all non-admin Konnect users have to log in through your IdP. Only the Konnect org owner can continue to log in with Konnect’s native authentication.
+
+{:.note}
+> This guide shows you how to configure SSO for Konnect with Microsoft Azure. See [Configure Azure IdP for Dev Portal](/konnect/dev-portal/access-and-approval/azure/) for those instructions.
+
+
+## Prerequisites
+### Create an application in Azure
+
+1. In [Azure](https://portal.azure.com/), navigate to **App registrations**. 
+
+1. Click **New registration** to register a new application:
+
+1. Name the application.
+
+1. Select **Accounts in this organizational directory only** for the **Supported account type**. 
+
+1. Select **Web** and enter the (ADD CORRECT URL FOR KONNECT HERE)
+    
+1. Save the application ID for later.
+
+
+### Configure the Azure application
+
+1. Click **New client secret**, enter a description, select an expiration value, and click **Add**.
+    
+    * Save the secret value for configuring {{site.konnect_short_name}}.
+
+
+1. Click **Overview** in the sidebar, then click the **Endpoints** tab.
+
+1. Copy the **OpenID Connect metadata document** URL and open it in your browser:
+
+1.  Your browser will display a large JSON blob object. In the object, find and save the `issuer` value.
+    
+    The `issuer` value will be used as the provider URL when configuring SSO in {{site.konnect_short_name}}.
+
+### Configure group claims in Azure
+
+Group claims automatically add or remove users from group memberships. To configure group claims, follow these steps: 
+
+1. On your new application page in Azure, click **Token configuration** in the sidebar.
+
+1. Click **+ Add groups claim** and do the following:
+    1. Select each checkbox in the **Select group types to include in Access, ID, and SAML tokens** section. 
+    1. Select **Group ID** for each section in **Customize token properties by type**.
+    1. Click **Add**. 
+
+    ![Group claim](/assets/images/docs/konnect/azure/group-claim.png)
+
+1. Click **Add optional claim**, select **ID** as the token type, and **email** as the claim.
+
+1. Click **Add**.
```

Comment:
question: i'm not sure if it matters but I was prompted to add a permission to the registered application in order to do this step. the `add` button never was available, after I added the permission it automatically submitted the form.

<img width="575" alt="Screenshot 2023-05-02 at 19 16 52" src="https://user-images.githubusercontent.com/7806510/235821521-a10ce726-d435-4b52-a073-f56a7942504f.png">

Reply:
@iamnande Did you have to go to a different page/area to add that permission? Just wondering if I should add those steps or if it should be a prerequisite (ex. you need the ___ permission).

Reply:
No I don't believe so (been a minute). When the claims are added and you go to slick save, this is the pop up that is essentially notifying you that because of the claim selection Azure is going to add a permission to the app on your behalf (I think).